Explorer/Assets/DCL/Audio/Systems/CalculateLandscapeAudioStatesJob.cs

```diff
@@ -0,0 +1,63 @@
+﻿using DCL.Audio.Systems;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Jobs;
+using Unity.Mathematics;
+
+namespace DCL.Audio.Jobs
+{
+    [BurstCompile]
+    public struct CalculateLandscapeAudioStatesJob : IJobParallelFor
```

Comment:
Now it's super clean, thank you 💚

Reply:
Im not 100% happy with it being initialized on the Landscape plugin, but it needs the TerrainGenerator initialized there and I'm not sure if I wanna create more dependencies between plugins or extract it outside in a container and feed it to both plugins... for now I think it will do, also as its very related with Landscape its also not necessarily so bad xD